spec/mailers/previews/order_preview.rb

```diff
@@ -0,0 +1,7 @@
+# Preview all emails at http://localhost:3000/rails/mailers/order
+class OrderPreview < ActionMailer::Preview
+  def order_delivered_email
+    order = Order.first
+    OrderMailer.with(order: order).order_delivered_email
```

Comment:
What happens if there will be no Orders in DB?

Reply:
It does not load order object and crashes with undefined method for nil. I have changed it to use factory to provide object